cmd/db.go

```diff
@@ -299,6 +300,7 @@ func init() {
 	resetFlags.String("db-url", "", "Resets the database specified by the connection string (must be percent-encoded).")
 	resetFlags.Bool("linked", false, "Resets the linked project with local migrations.")
 	resetFlags.Bool("local", true, "Resets the local database with local migrations.")
+	resetFlags.Bool("skip-seed", false, "Skip running the seed script after reset")
```

Comment:
```suggestion
	resetFlags.Bool("no-seed", false, "Skip running the seed script after reset")
```

Let's call this `--no-seed` for consistency with other flags.